pkg/security/ptracer/cws.go

```diff
@@ -336,29 +336,32 @@ func StartCWSPtracer(args []string, grpcAddr string, verbose bool) error {
 	logDebugf("Run %s %v [%s]\n", entry, args, os.Getenv("DD_CONTAINER_ID"))
 
 	var (
-		client proto.SyscallMsgStreamClient
+		client net.Conn
 	)
 
 	// GRPC
 	if grpcAddr != "" {
-		conn, err := grpc.Dial(grpcAddr, grpc.WithTransportCredentials(insecure.NewCredentials()))
+		tcpAddr, err := net.ResolveTCPAddr("tcp", grpcAddr)
```

Comment:
nit but the the `grpcAddr` variable should be renamed and the command line argument as well